Tests/StarknetTests/Data/TypedDataTests.swift

```diff
@@ -456,4 +456,22 @@ final class TypedDataTests: XCTestCase {
             XCTAssertEqual(hash, expectedResult)
         }
     }
+
+    func testEncodingTimestamp() throws {
+        let tdJsonStr = try loadTypedDataJsonStringFromFile(name: "typed_data_rev_1_timestamp_example")
+        guard let tdEncodedContents = tdJsonStr.data(using: .utf8) else {
+            throw DecodingError.dataCorrupted(DecodingError.Context(codingPath: [], debugDescription: "Failed to load typed data"))
+        }
+
+        do {
+            let td = try JSONDecoder().decode(StarknetTypedData.self, from: tdEncodedContents)
+            let tdEncoded = try JSONEncoder().encode(td)
+            let originalJson = try JSONSerialization.jsonObject(with: tdEncodedContents, options: []) as? [String: Any]
+            let encodedJson = try JSONSerialization.jsonObject(with: tdEncoded, options: []) as? [String: Any]
+
+            XCTAssertEqual(originalJson as NSDictionary?, encodedJson as NSDictionary?)
+        } catch let e {
+            throw e
+        }
+    }
```

Comment:
Refactor + extending this tests will be covered in https://github.com/software-mansion/starknet.swift/issues/208